VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/wait-for-hearing-panel/wait-for-hearing-panel.component.html

```diff
@@ -0,0 +1,71 @@
+<div *ngIf="currentTime" class="govuk-panel" [ngClass]="getCurrentTimeClass()">
+  <div class="govuk-body govuk-!-margin-bottom-0" [ngClass]="getCurrentTimeClass()">
+    <div *ngIf="!isWitnessOrHasWitnessLink && !isJoh && hearing.isOnTime()">
+      <h2 class="govuk-heading-m white">{{ 'waiting-room.please-wait' | translate }}</h2>
+      <p class="white">{{ 'waiting-room.hearing-starting-at' | translate }} {{ hearing.scheduledStartTime | date: 'HH:mm' }}</p>
+    </div>
+    <div *ngIf="isJoh && (hearing.isOnTime() || hearing.isStarting() || hearing.isDelayed())">
+      <h2 class="govuk-heading-m white">{{ 'waiting-room.please-wait' | translate }}</h2>
+      <p class="white">
+        {{ 'waiting-room.hearing-starting-at' | translate: { date: hearing.scheduledStartTime | date: 'HH:mm' } }}
+      </p>
+    </div>
+
+    <div *ngIf="!isWitnessOrHasWitnessLink && !isJoh && hearing.isStarting()">
+      <h2 class="govuk-heading-m white">{{ 'waiting-room.hearing-status-messages.starting-title' | translate }}</h2>
+      <p class="white">{{ 'waiting-room.hearing-status-messages.starting-body' | translate }}</p>
+    </div>
+
+    <div *ngIf="!isWitnessOrHasWitnessLink && !isJoh && hearing.isDelayed()">
+      <h2 class="govuk-heading-m">{{ 'waiting-room.hearing-status-messages.delayed-title' | translate }}</h2>
+      <p class="govuk-body">{{ 'waiting-room.hearing-status-messages.delayed-body' | translate }}</p>
+    </div>
+
+    <div *ngIf="!isWitnessOrHasWitnessLink && !isJoh && hearing.isClosed()">
+      <h2 class="govuk-heading-m white">{{ 'waiting-room.hearing-status-messages.closed' | translate }}</h2>
+    </div>
+
+    <div *ngIf="!isWitnessOrHasWitnessLink && !isJoh && hearing.isPaused()" class="hearing-title-mt--20">
+      <p class="white">{{ 'waiting-room.hearing-status-messages.paused' | translate }}</p>
+    </div>
+
+    <div *ngIf="hearing.isSuspended()">
+      <h2 class="govuk-heading-m">{{ 'waiting-room.hearing-status-messages.suspended-title' | translate }}</h2>
+      <p class="govuk-body">{{ 'waiting-room.hearing-status-messages.suspended-body' | translate }}</p>
+    </div>
+
+    <div *ngIf="(isWitnessOrHasWitnessLink || isQuickLinkUser) && hearing.isInSession()">
+      <h2 class="govuk-heading-m white">{{ 'waiting-room.hearing-status-messages.in-session-title' | translate }}</h2>
+      <p class="white">
+        {{ 'waiting-room.hearing-status-messages.in-session-body' | translate }}
+      </p>
+    </div>
+
+    <div *ngIf="isWitnessOrHasWitnessLink && (hearing.isOnTime() || hearing.isStarting() || hearing.isDelayed())">
+      <h2 class="govuk-heading-m white">{{ 'waiting-room.please-wait' | translate }}</h2>
+      <p class="white">
+        {{ 'waiting-room.hearing-about-to-start' | translate }}
+      </p>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+      <div class="govuk-panel__body govuk-!-margin-top-4">
+        <app-analogue-clock
```

Comment:
bit of a major delay before this component shows. it should be one of the first to load.